resources/stubs/modules/routes.stub

```diff
@@ -1,11 +1,11 @@
 <?php
 
-Route::get('/', '$STUDLY_NAME$Controller@index');
+Route::get('/', 'IndexController@index');
```

Comment:
Can't remember, did I name the default one to `IndexController` in the config file?

Reply:
Umm yes, by default ut created IndexController.php